app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -535,6 +536,14 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testnextArrowDescription() {
```

Comment:
The naming of the test case it incorrect.
Tests should follow this naming convention:
https://github.com/oppia/oppia-android/wiki/Oppia-Android-Testing#naming-convention

So a better name would be 
`testOnboardingFragment_nextArrowIcon_hasCorrectContentDescription`

Reply:
Also, this test should come above `scrollToPosition` function.

Reply:
Got it...both changes done.